cpp/daal/include/services/daal_defines.h

```diff
@@ -65,6 +66,8 @@
 #if !defined(DAAL_INT)
     #if defined(_WIN64) || defined(__x86_64__)
         #define DAAL_INT __int64
+    #elif defined(__aarch64__)
```

Comment:
I see inonsistency in macro usage. Like `defined(__aarch64__)` vs `defined(__ARM_ARCH)`